packages/babel-plugin/src/styled/index.ts

```diff
@@ -35,7 +35,7 @@ const extractStyledDataFromTemplateLiteral = (
   if (
     t.isMemberExpression(node.tag) &&
     t.isIdentifier(node.tag.object) &&
-    node.tag.object.name === meta.state.compiledImports?.styled &&
+    meta.state.compiledImports?.styled?.includes(node.tag.object.name) &&
```

Comment:
out of curiosity, why did you change all of these? is the latter easier to read?

Reply:
No, because they're arrays now instead of strings 😃

Reply:
riiiiight, I saw the types now 😅